modules/images/dominant-color/class-wp-dominant-color.php

```diff
@@ -0,0 +1,383 @@
+<?php
+/**
+ *  Add Dominant Color preload support to WordPress.
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Add Dominant Color preload support to WordPress.
+ *
+ * @since 1.0.0
+ */
+class WP_Dominant_Color {
```

Comment:
While I understand the desire for a more object-oriented implementation, we need to work towards an eventual WordPress core merge, which means mostly procedural code.

I'm of course not saying there cannot be any classes, but we should follow WordPress's usage of classes. The entry points for functionality are almost exclusively functions that are hooked in. They may use classes (e.g. `WP_Post_Type` or `WP_Post`), but the overall implementation is function-based.

Reply:
OK I will remove the class